container/secrets.go

```diff
@@ -81,5 +86,8 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	if _, err := os.Stat(overrideDir); err == nil {
```

Comment:
So we really need a union of the files in both directories with /etc overriding content in /usr/share.